src/graph/mod.rs

```diff
@@ -11,11 +11,13 @@ mod disjoint_set_union;
 mod floyd_warshall;
 mod graph_enumeration;
 mod heavy_light_decomposition;
+mod kosaraju;
 mod lowest_common_ancestor;
 mod minimum_spanning_tree;
 mod prim;
 mod prufer_code;
 mod strongly_connected_components;
+mod tarjans_ssc;
```

Comment:
Shouldn't it be `tarjans_scc`?

Reply:
Fixed the typo error `tarjans_ssc` -> `tarjans_scc`